app/presentation/ShareItem/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text, View } from 'react-native';
+import { RectButton } from 'react-native-gesture-handler';
+
+import Avatar from '../../containers/Avatar';
+import styles, { ROW_HEIGHT } from './styles';
+import DisclosureIndicator from '../../containers/DisclosureIndicator';
+
+export { ROW_HEIGHT };
+
+const ShareItem = React.memo(({
```

Comment:
I think we should reuse UserItem component.
We'll have to change it a little to accept channels and use RectButton.